src/scss/components/selector-strip/_hxSelector.scss

```diff
@@ -56,41 +59,53 @@
       // unchecked (focused)
       &:focus + span {
         background-color: $cyan-100;
+        background-color: var(--hxSelector-unchecked-bgcolor, $cyan-100);
         color: $cyan-900;
+        color: var(--hxSelector-unchecked-color, $cyan-900);
       }
 
       // checked (unfocused)
       &:checked + span {
         background-color: $cyan-900;
+        background-color: var(--hxSelector-checked-bgcolor, $cyan-900);
         color: $gray-0;
+        color: var(--hxSelector-checked-color, $gray-0);
       }
 
       // checked (focused)
       &:checked:focus + span {
         box-shadow: 0 0 4px 0 $cyan-900;
+        box-shadow: 0 0 4px 0 var(--hxSelector-checked-focus-boxShadow, $cyan-900);
       }
 
       // pressed (checked or unchecked)
       &:active + span {
         background-color: $cyan-700;
+        background-color: var(--hxSelector-active-bgcolor, $cyan-700);
         color: $gray-0;
+        color: var(--hxSelector-active-color, $gray-0);
       }
 
       // invalid (unchecked)
       &[invalid] + span {
         background-color: $gray-0;
+        background-color: var(--hxSelector-invalid-bgcolor, $gray-0);
         border: 2px solid $red-status-500;
+        border: 2px solid var(--hxSelector-invalid-border, $red-status-500);
         color: $red-status-500;
+        color: var(--hxSelector-invalid-color, $red-status-500);
       }
 
       // invalid (checked)
       &[invalid]:checked + span {
         background-color: $red-100;
+        background-color: var(--hxSelector-invalid-checked-bgcolor, $red-100);
```

Comment:
Please use updated  `v2.0` colors to replace `$red-XXX`.

Reply:
Changes have been done, please review.